### ISSUE TITLE
Memory issue when evaluating the dataset

### ISSUE THREAD

**Issue by Evm7** [Droliven/MSRGCN#12]
Hi! Thanks for your nice work.

Given the current new tendency of Human Motion Forecasting in Human3.6M evaluates the model in the test set based on 256 samples (not all or 8, as you provide), I wanted to get these results. However, my PC does not have enough memory to run the evaluation.
To solve this issue, it would be enough to only load the test set, rather than loading first the training set to get the global_max and global_min values. Could you please provide the values for global_max and global_min with the default configuration set?
